native-messaging/check_config_win.py

```diff
@@ -0,0 +1,64 @@
+'''
+A Python script that attempts to check that the configuration of your
+nativeMessaging app is set up correctly
+
+Currently requires Python 3.
+
+If you find more issues with setting this up, let's see if we can add to this
+script.
+'''
+import json
+import os
+import re
+import winreg
+
+key_path = 'Software\\Mozilla\\NativeMessagingHosts\\ping_pong'
+# Assuming current user overrides local machine.
+key_roots = ['HKEY_CURRENT_USER', 'HKEY_LOCAL_MACHINE']
+
+found_key = False
+
+for root in key_roots:
+    key = winreg.OpenKey(getattr(winreg, root), key_path)

```

Comment:
Using python 3.x I get this info. 
```
PS D:\git\webextensions-examples\native-messaging> python .\check_config_win.py
Traceback (most recent call last):
  File ".\check_config_win.py", line 22, in <module>
    key = winreg.OpenKey(getattr(winreg, root), key_path)
FileNotFoundError: [WinError 2] Das System kann die angegebene Datei nicht finden
```

which is ok. Since I didn't create a registry key yet. .... BUT IMO the error message is misleading.

What I would expect is something similar to: 

```
Please create a registry key as described in: https://developer.chrome.com/extensions/nativeMessaging#native-messaging-host-location  ... BUT ... use the key_path XXXXXX
```

Or something that works. ... The problem I have atm ... I don't have an idea, what's really needed! and the example docs doesn't help. .. and the link: https://developer.chrome.com/extensions/nativeMessaging#native-messaging-host-location is kind of crap too. since it contains chrome info and not FF info.

Reply:
Did you see: https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Native_manifests#Manifest_location?

Reply:
nope. thx

Reply:
But still. I think that link should be returned then :)

Reply:
Please open an issue on this repo so then a contributor can maybe fix it.